src/hooks/v2/contractReader/BallotState.ts

```diff
@@ -0,0 +1,14 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { BallotState } from 'models/ballot-state'
+
+import { V2ContractName } from 'models/v2/contracts'
+
+import useV2ContractReader from './V2ContractReader'
+
+export function useBallotState(projectId: BigNumber | undefined) {
+  return useV2ContractReader<BallotState>({
+    contract: V2ContractName.JBFundingCycleStore,
```

Comment:
@johnnyd-eth just a note, we for `useV2ContractReader` we can use the `V2ContractName` instead of the actual contract object (`contracts?.JBFundingCycleStore`). Simplifies things a bit.

Reply:
ahh true ok got it